packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx

```diff
@@ -245,13 +245,13 @@ const GivingSeasonHeader = ({
       href: '/voting-portal/select-candidates',
     },
     {
-      label: '2. Compare',
+      label: '2. Comparison tool',
```

Comment:
I much prefer "Compare" because all the others are verbs whereas this is a noun, I think it makes it really stand out. I'm going to change it back to "Compare" but someone can override me if they feel strongly